src/renderer/store/modules/subscriptions.js

```diff
@@ -69,19 +69,21 @@ const actions = {
 }
 
 const mutations = {
-  updateVideoCacheByChannel(state, { channelId, videos }) {
+  updateVideoCacheByChannel(state, { channelId, videos, timestamp }) {
     const existingObject = state.videoCache[channelId]
     const newObject = existingObject ?? { videos: null }
     if (videos != null) { newObject.videos = videos }
+    if (timestamp != null) newObject.timestamp = timestamp
```

Comment:
Not suggesting change to mutation but about `updateSubscriptionVideosCacheByChannel`
Why not just make `updateSubscriptionVideosCacheByChannel` pass `timestamp: timestamp: new Date()` into `updateVideoCacheByChannel` (There is no use case for passing non `new Date()` into `updateSubscriptionVideosCacheByChannel`)

Same for other methods

See example at https://github.com/FreeTubeApp/FreeTube/blob/v0.20.0-beta/src/renderer/store/modules/playlists.js#L63-L64

Reply:
I was thinking about making that the default value if none was passed in. I don't really have a preference now that it's already all implemented. Give this reply a thumbs up and I'll do it.

Reply:
I am fine with default value approach (though in what use case a non current time is needed...?)
Why not just thumbs up? Coz that doesn't send a notification

Reply:
For the first Q, I dunno. We have similar usecases with functions like `updateLastCommunityRefreshTimestampByProfile` where we pass a `minTimestamp`, so you never know. For the second Q, I'm fine with checking the tab later; I just don't want anyone to feel like they're being demanded.